<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLCDNumber" name="workMinutes">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="breakMinutes">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>70</y>
      <width>141</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Work minutes</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>40</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Break minutes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="doWork">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>220</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>do Work</string>
    </property>
   </widget>
   <widget class="QPushButton" name="doBreak">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>220</y>
      <width>97</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>do Break</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pause">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>360</y>
      <width>97</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reset">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>400</y>
      <width>97</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="breakMinutesCost">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>260</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
    <property name="value">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>270</y>
      <width>231</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Work minutes per break minutes</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="breakMinutesThreshold">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>300</y>
      <width>60</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>90</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>300</y>
      <width>221</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Break minutes collect threshold</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Work seconds</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="workSeconds">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="breakSeconds">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>70</y>
      <width>141</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>40</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Break seconds</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
